### PR TITLE
Chromatic CI

### PR DESCRIPTION
**Pull request by mcrascal**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -75,6 +75,7 @@
 		"dev:test-env": "pnpm --filter ./sites/test-env dev",
 		"dev:core-components": "pnpm --filter ./packages/ui/core-components package:watch",
 		"sources:example-project": "pnpm --filter ./sites/example-project sources",
+		"sources:core-components": "pnpm --filter ./packages/ui/coure-components sources",
```
Comment (ItsMeBrianD) [inline package.json:78]:
```suggestion
		"sources:core-components": "pnpm --filter ./packages/ui/core-components sources",
```

**Comment by ItsMeBrianD** [comment]
@mcrascal what is currently blocking this?

We should enable the diffs in PRs before merging this as well

**Comment by mcrascal** [comment]
@ItsMeBrianD I think we're almost good to go 

I'm going to pull in the latest on query debugger and see if I can make it fit. 

The other item on my list is the TW config -- I would really like to get that syncing into the storybook before we set a bunch of baselines, but I won't hold it if I can't get that working.

**Comment by mcrascal** [comment]
@ItsMeBrianD I think we're good to go

**Comment by ItsMeBrianD** [comment]
I think there's one more issue here - it looks like chromatic isn't running sources when building storybook. We probably need to have `npm run sources && npx chromatic ...` in the package.json

File: packages/lib/sdk/src/usql/query/Query.js [hunk 2]
```diff
@@ -868,7 +892,9 @@ DESCRIBE ${this.text.trim()}
 			);
 
 		Query.#constructing = true;
+		console.log(opts.id);
```
Comment (ItsMeBrianD) [inline packages/lib/sdk/src/usql/query/Query.js:895]:
```suggestion
```

**Review by ItsMeBrianD — APPROVED** [review]

**Pull request merged**